python_modules/dagster/setup.py

```diff
@@ -167,7 +167,7 @@ def get_version() -> str:
             "types-toml",  # version will be resolved against toml
         ],
         "ruff": [
-            "ruff==0.5.1",
+            "ruff==0.5.4",
```

Comment:
we like to do these things in separate PRs; if you'd like to bump the ruff version I suggest you put up a separate PR for that! Although we usually handle these types of version bumps internally.